### PR TITLE
[ti_crowdstrike] Fix intel pagination bug due to cursor formatting

### PR DESCRIPTION
**Pull request by chrisberkhout**
## Proposed commit message

```
[ti_crowdstrike] Fix intel pagination bug caused by cursor formatting

Integer numbers of 1 million or more should be cast to int before
they're cast to string so that they are not formatted with an exponent.

The second example below shows how the timestamp used in the intel
filter condition was being misformatted:

  mito -data <(echo '{"n": 1725646291}') <(echo 'state.n')
  # => 1725646291

  mito -data <(echo '{"n": 1725646291}') <(echo 'string(state.n)')
  # => "1.725646291e+09"

  mito -data <(echo '{"n": 1725646291}') <(echo 'string(int(state.n))')
  # => "1725646291"
```

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/ti_crowdstrike/data_stream/intel/agent/stream/cel.yml.hbs [hunk 1]
```diff
@@ -67,13 +67,13 @@ program: |
       {
         "events": {
           "error": {
-            "code": string(resp.StatusCode),
+            "code": string(int(resp.StatusCode)),
```
Comment (efd6) [inline packages/ti_crowdstrike/data_stream/intel/agent/stream/cel.yml.hbs:70]: I don't think this needs to happen for number that are as small as status codes.

File: packages/ti_crowdstrike/data_stream/intel/agent/stream/cel.yml.hbs [hunk 2]
```diff
@@ -39,9 +39,9 @@ program: |
         "/intel/combined/indicators/v1?" + {
           "sort": ["_marker.asc"],
           "offset": ["0"],
-          "limit": [string(state.batch_size)],
+          "limit": [string(int(state.batch_size))],
```
Comment (efd6) [inline packages/ti_crowdstrike/data_stream/intel/agent/stream/cel.yml.hbs:42]: I would imagine that batch sizes are also under the e-notation limit too.

Comment (chrisberkhout) [reply packages/ti_crowdstrike/data_stream/intel/agent/stream/cel.yml.hbs:42]: Yeah. The limit is >= 1,000,000.
Removed.

File: packages/ti_crowdstrike/data_stream/intel/agent/stream/cel.yml.hbs (hunk 1 above)
Comment (chrisberkhout) [reply packages/ti_crowdstrike/data_stream/intel/agent/stream/cel.yml.hbs:70]: Removed

**Review by efd6 — APPROVED** [review]

**Pull request merged**